src/pathwayvis/services/mapoption.service.ts

```diff
@@ -438,6 +453,12 @@ export class MapOptionService {
         event_label: id,
       });
     }
+    if (action.type === 'reaction:minimize:do') {
```

Comment:
Rather use an else if operator, or with the consistent naming, you could just decompose the action type.